bugbug/models/performancebug.py

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+
+import xgboost
+from imblearn.over_sampling import BorderlineSMOTE
+from imblearn.pipeline import Pipeline as ImblearnPipeline
+from sklearn.compose import ColumnTransformer
+from sklearn.feature_extraction import DictVectorizer
+from sklearn.pipeline import Pipeline
+
+from bugbug import bug_features, bugzilla, feature_cleanup, utils
+from bugbug.model import BugModel
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def _is_performance_bug(bug_data) -> bool:
+    keywords = bug_data.get("keywords", [])
+    whiteboard = bug_data.get("whiteboard", "").lower()
+    cf_performance = bug_data.get("cf_performance", "")
+
+    performance_keywords = ("perf", "topperf", "main-thread-io")
+
+    whiteboard_performance_text = (
+        "fxperf",
+        "fxperfsize",
+        "snappy",
+        "pdfjs-c-performance",
+        "pdfjs-performance",
+        "sp3",
+    )
+
+    if any(
+        keyword.startswith(performance_keyword)
+        for keyword in keywords
+        for performance_keyword in performance_keywords
+    ):
+        return True
+
+    if any(f"[{text}" in whiteboard for text in whiteboard_performance_text):
+        return True
+
+    if cf_performance and cf_performance not in ("---", "?"):
```

Comment:
Since we're using the `bug_to_types` function from `models/bugtype.py` to find the bug type, is it okay to change the logic so that bugs with `cf_performance == "none"` aren't seen as performance bugs?

https://github.com/mozilla/bugbug/blob/d220467520ba7416425e94e530f4f774832967e5/bugbug/models/bugtype.py#L69-L72

Reply:
Yeah, let's do that for now, if it is "none" it should not be considered as a perf bug.

We can file a follow-up issue to investigate using these as performance or not as performance in the training set but excluding them from the test set to see if they have any impact.

Reply:
I filed a follow-up issue: https://github.com/mozilla/bugbug/issues/3901